### DISCUSSION TITLE
"latest" image on Docker Hub isn't

### DISCUSSION THREAD

**Discussion by ipilcher** [LMS-Community/slimserver-platforms#36] [General]
The image tagged as `latest` on [Docker Hub](https://hub.docker.com/r/lmscommunity/logitechmediaserver/tags) appears to actually be 8.2.0.  Not sure what version it's supposed to be, but I'm pretty sure that's not it.

**Comment by michaelherger** [comment]
Please see the descriptions of `latest`, `stable`, and `dev` on https://hub.docker.com/r/lmscommunity/logitechmediaserver. 8.2.0 indeed is the latest released version.

**Reply by ipilcher** [reply]
Thanks for clarifying.  (It still seems super-weird to me that `latest` is older than `stable`, but I'm very new to this whole container thing.)

**Reply by michaelherger** [reply]
`stable` is basically `latest` official release, with a few bug fixes, which haven't been released yet. But the system is considered stable, as only bug fixes are applied, no new features or bigger changes. While it's inspired by Debian's stable/testing/unstable scheme, we are using `latest` here, as in the Docker environment this often is the build that a developer expects users to use.
